unumpy/_multimethods.py

```diff
@@ -1421,3 +1565,209 @@ def _vander_default(x, N=None, increasing=False):
 @all_of_type(ndarray)
 def vander(x, N=None, increasing=False):
     return (x,)
+
+
+@create_numpy(_self_argreplacer)
+@all_of_type(ndarray)
+def split(ary, indices_or_sections, axis=0):
+    return (ary,)
+
+
+@create_numpy(_self_argreplacer)
+@all_of_type(ndarray)
+def array_split(ary, indices_or_sections, axis=0):
+    return (ary,)
+
+
+@create_numpy(
+    _self_argreplacer,
+    default=lambda ary, indices_or_sections: split(ary, indices_or_sections, axis=2),
+)
+@all_of_type(ndarray)
+def dsplit(ary, indices_or_sections):
+    return (ary,)
+
+
+@create_numpy(
+    _self_argreplacer,
+    default=lambda ary, indices_or_sections: split(ary, indices_or_sections, axis=1),
+)
+@all_of_type(ndarray)
+def hsplit(ary, indices_or_sections):
+    return (ary,)
+
+
+@create_numpy(
+    _self_argreplacer,
+    default=lambda ary, indices_or_sections: split(ary, indices_or_sections, axis=0),
+)
+@all_of_type(ndarray)
+def vsplit(ary, indices_or_sections):
+    return (ary,)
+
+
+def _tile_default(A, reps):
+    if ndim(reps) == 0:
+        reps = (reps,)
+
+    prep = len(reps) - ndim(A)
+    if prep > 0:
+        for _ in range(prep):
+            A = expand_dims(A, 0)
+    elif prep < 0:
+        reps = tuple(itertools.repeat(1, abs(prep))) + reps
+
+    for axis, dim in enumerate(reps):
+        if not isinstance(dim, numbers.Integral):
+            dim = dim[()]
```

Comment:
What's going on here?

Reply:
This was added be @hameerabbasi to fix a failing test in the Sparse backend. It gets the value from a zero-dimensional array.

I would move this to the beginning of the function, like this:

```python
def _tile_default(A, reps):
    if ndim(reps) == 0:
        reps = (reps[()],)
```

Reply:
`reps` probably shouldn't be marked as `ndarray`, that would avoid the issue of 0dim arrays. In fact cupy breaks if it is an array since they don't allow `reps` to be on the GPU.

Reply:
Don't dispatch `reps` then?

Reply:
Yeah

Reply:
Even if `reps` is not dispatched the default should behave correctly if an array is passed, right? Because internally the default converts `reps` to an array with `concatenate`. Is this desirable or not?

Reply:
It is definitely not desirable, as I said it won't work on CuPy. Instead, you could do `((1,) * -n) + tuple(reps)`.